release-notes/6.0/preview/api-diff/preview5/.Net/6.0-preview5_System.Runtime.InteropServices.ObjectiveC.md

```diff
@@ -0,0 +1,24 @@
+# System.Runtime.InteropServices.ObjectiveC
+
+``` diff
++namespace System.Runtime.InteropServices.ObjectiveC {
```

Comment:
@AaronRobinsonMSFT, it would be good to add this to https://github.com/dotnet/core/issues/6099, I think

Reply:
Great idea! Thanks.

Reply:
https://github.com/dotnet/core/issues/6099#issuecomment-855058837